homeworks/hw1/Daniil_Sidorov/main.py

```diff
@@ -0,0 +1,145 @@
+import argparse
+
+import pandas as pd
+import torch
+import torch.nn as nn
+from sklearn.metrics import accuracy_score, confusion_matrix
+from sklearn.preprocessing import StandardScaler
+from sklearn.metrics import classification_report
+import matplotlib.pyplot as plt
+
+train_csv = pd.read_csv('../data/train.csv')
```

Comment:
Use either argparse to configure the CSV path, or hardcode them as you do here, but not both